File: services/Microservice.MapsPointsUser.Api/Model/Dto/PoiDto.cs
namespace Microservice.MapsPointsUser.Api.Model.Dto
{
    public class PoiDto
    {
        public Guid PoiId { get; set; }
        public Guid? BusinessId { get; set; }
        public string Name { get; set; }
        public string Category { get; set; }
        public string Description { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public bool IsActive { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}

File: mobile-android/AndandoAndo_App/AndandoAndo_App/app/src/main/java/com/example/app_andando_ando/data/dto/mapUser/PoiDto.kt
package com.example.app_andando_ando.data.dto.mapUser

data class PoiDto(
    val poiId: String,
    val businessId: String?,
    val name: String,
    val category: String?,
    val description: String?,
    val latitude: Double,
    val longitude: Double,
    val imageUrl: String?,
    val menu: List<MenuItemDto>?,
    val openingHours: List<OpeningHourDto>?,
    val averageRating: Double?,
    val isActive: Boolean,
    val imageLocalPath: String?
)

data class MenuItemDto(
    val menuItemId: String,
    val name: String,
    val description: String?,
    val price: Double
)

data class OpeningHourDto(
    val dayOfWeek: Int,
    val open: String,
    val close: String
)
